src/components/AssetsImagePreviewFilter/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CheckBox } from '@edx/paragon';
+
+import WrappedMessage from '../../utils/i18n/formattedMessageWrapper';
+import messages from './displayMessages';
+
+const AssetsImagePreviewFilter = ({ updateImagePreview, isImagePreviewEnabled }) => (
+  <CheckBox
+    // key={type.key} // I don't know if we need this. It's only for lists of elements, right?
```

Comment:
I'm not sure what these three values should be. I copied the CheckBox from the AssetsFilters. I don't think we need key, but I'm not sure about id and name.